webcompat/templates/home-page/form.html

```diff
@@ -15,9 +15,25 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
       <div class="r-Grid r-Grid--withGutter">
         <div class="r-Grid-cell r-minM--1of2">
           <div class="wc-Form-group js-Form-group">
+            <div class="wc-Form-information">
+              {{ form.url.label(class_='wc-Form-label') }}
+              {% if form.url.errors %}
+                {% for error in form.url.errors %}
+                  <span class="wc-Form-helpMessage">{{ error }}</span>
+                {% endfor %}
+              {% endif %}
+            </div>
+            <span class="wc-Form-validation">
+            {{ form.url(class_='wc-Form-item required',
+               placeholder='e.g. https://example.com',
+               required=True, type='url', pattern='') }}
+            </span>
+          </div>
+
+          <div class="wc-Form-group js-Form-group">
             <fieldset class="{% if form.problem_category.errors %}is-error{% endif %}">
               <div class="wc-Form-information">
-                <label class="wc-Form-label">What seems to be the trouble? <span class="wc-Form-required">*</span></label>
+                <label class="wc-Form-label" for="problem_type">Type of bug <span class="wc-Form-required">*</span></label>
```

Comment:
@cch5ng understood. 

Ah simple. It's because two different names are being used in the logic in the code `problem_category` and `problem_type` if the same name is used everywhere. There should be no issue. 

As an experiment I did this in form.html

```
                {{ form.problem_category.label(class_='wc-Form-label') }}
                <label class="wc-Form-label" for="problem_type">Type of bug <span class="wc-Form-required">*</span></label>
```

with the following in form.py

```python
problem_category_label = (u'Type of bug'
                          u'<span class="wc-Form-required">*</span>')

# …

    problem_category = RadioField(problem_category_label,
                                  [InputRequired(message=radio_message)],
                                  choices=problem_choices)

```

which resulted in the html as:

```html
<div class="wc-Form-information">
                <label class="wc-Form-label" for="problem_category">Type of bug<span class="wc-Form-required">*</span></label>
                <label class="wc-Form-label" for="problem_type">Type of bug <span class="wc-Form-required">*</span></label>
                
              </div>
```

The only difference here in between the hard coded value and the programmatically generated is indeed the `for` attribute value, but this comes from the chosen variable name in form.py. This can be changed. 

So either we switch everything to `problem_category` or everything to `problem_type` but mixing both seems not a good idea as we noticed. :)

Searching for `problem type`.

```
→ git rg problem_type
CHANGELOG.md
124:* Add support for a problem_type param to preselect type in form [Pull #1362](https://github.com/webcompat/webcompat.com/pull/1362)

tests/functional/reporting-non-auth.js
293:      "problem_type param selects problem type": function() {
296:          url("/issues/new?problem_type=video_bug"),

webcompat/form.py
218:        'problem_type': get_radio_button_label(
229:**Problem type**: {problem_type}

webcompat/static/js/lib/bugform.js
20:    problem_type: {
61:  this.problemType = this.inputs.problem_type.el;
175:    // If we have a problem_type param, and it matches the value, select it for
178:    var problemType = location.href.match(/problem_type=([^&]*)/);
222:      this.makeInvalid("problem_type");
224:      this.makeValid("problem_type");
354:      case "problem_type":
392:      this.inputs["problem_type"].valid &&

webcompat/templates/home-page/form.html
36:                <label class="wc-Form-label" for="problem_type">Type of bug <span class="wc-Form-required">*</span></label>
```

Searching for `problem_category`

```
→ git rg problem_category
tests/functional/reporting-non-auth.js
132:            .findByCssSelector("#problem_category-0")
272:            .findByCssSelector("#problem_category-0")

tests/test_urls.py
128:        data = {'problem_category': u'mobile_site_bug',

webcompat/form.py
83:    problem_category = RadioField([InputRequired(message=radio_message)],
205:    problem_summary = get_problem_summary(form_object.get('problem_category'))
219:            form_object.get('problem_category'), problem_choices),

webcompat/static/js/lib/bugform.js
21:      el: $("[name=problem_category]"),
221:    if (!$("[name=problem_category]:checked").length) {

webcompat/templates/home-page/form.html
34:            <fieldset class="{% if form.problem_category.errors %}is-error{% endif %}">
37:                {% if form.problem_category.errors %}{% for error in form.problem_category.errors %}
41:                {% for subradio in form.problem_category %}

webcompat/views.py
172:        must_parameters = set(['url', 'problem_category', 'description',
```

I think we have a hard dependency on `problem_type` because of the `type-media` people sending us a URI with `problem_type` in parameters.

So I proposed we switch to from `problem_category` to `problem_type`. @miketaylr opinion?